source-boilerplate/testing/testing.go

```diff
@@ -284,6 +284,9 @@ func (a *pullAdapter) Send(m *pc.Response) error {
 			var binding string
 			if idx := doc.Binding; int(idx) < len(a.bindings) {
 				binding = string(a.bindings[idx].Collection.Name)
+				if len(binding) == 0 {
```

Comment:
I created the tests for `source-nativehello` as part of the great protocol refactoring from several weeks ago primarily as a means for verifying that the boilerplate changes worked. At this point I don't think the `source-nativehello` tests are very valuable anymore - we have many other connectors that run tests which exercise the boilerplate code paths. So I think I'd rather just remove `source-hello-world/main_test.go` and back this change out here that accounts for a capture spec with zero bindings since it only applies for those tests.